### PR TITLE
Ar5 ccf gwps

### LINKED ISSUE #22: AR5 CCF GWPs
**Is your feature request related to a problem? Please describe.**

I need AR5 GWPs with climate-carbon feedbacks for PRIMAP2 (which uses openscm-units)

**Describe the solution you'd like**

I forked openscm-units and will add the AR5CCF GWPs. I will make a pull request once I'm done. Does that work for you?

### PR DESCRIPTION
**Pull request by JGuetschow**
filles missing AR5 GWPs

# Pull request

Please confirm that this pull request has done the following:

- [x] Tests added
- [ ] Documentation added (where applicable)
- [ ] Example added (either to an existing notebook or as a new notebook, where applicable)
- [x] Description in ``CHANGELOG.rst`` added

closes #22 

## Adding to CHANGELOG.rst

Please add a single line in the changelog notes similar to one of the following:

```
- (`#XX <https://github.com/openscm/openscm-units/pull/XX>`_) Added feature which does something
- (`#XX <https://github.com/openscm/openscm-units/pull/XX>`_) Fixed bug identified in (`#YY <https://github.com/openscm/openscm-units/issues/YY>`_)
```

### CODE REVIEW

**Review by mikapfl — APPROVED** [review]
I checked some oddities and found an error in the PR and an error in the data before the PR. The rest looks good to me!

File: src/openscm_units/data/metric_conversions.csv [hunk 1]
```diff
@@ -1,92 +1,92 @@
...
+HCFE235da2,350,491,491,595
+HFE245cb2,708,654,654,790
+HFE245fa2,659,812,812,981
+HFE347mcc3,575,530,530,641
+HFE347pcf2,580,889,889,1072
+HFE356pcc3,110,413,413,500
+HFE449sl,297,421,421,509
+HFE569sf2,59,57,57,69
+HFE4310pccc124,1870,2820,2820,3353
+HFE236ca12,2800,5350,5350,6260
```
[the remark this answers is not in the document]
Comment (JGuetschow) [reply src/openscm_units/data/metric_conversions.csv:73]: OK, I thought someone had only filled values for ar5 which had changed. Thus I filled the rest. But it makes more sense if the values were in the wrong column.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -3,7 +3,7 @@ Changelog
 
 master
 ------
-
+- (`#23 <https://github.com/openscm/openscm-units/pull/23>`_) Add AR5 GWPs with climate-carbon feedbacks (closes `#22 <https://github.com/openscm/openscm-units/issues/22>`_)
```
Comment (znicholls) [inline CHANGELOG.rst:6]:
```suggestion
- (`#23 <https://github.com/openscm/openscm-units/pull/23>`_) Add AR5 GWPs with climate-carbon cycle feedbacks (closes `#22 <https://github.com/openscm/openscm-units/issues/22>`_)
```

File: src/openscm_units/data/metric_conversions.csv [hunk 3]
```diff
@@ -1,92 +1,92 @@
...
-HFE365mcf3,,,
-HFE374pc2,,,627
-PFPMIE,,10300,9710
-CHCl3,4,,16
-CH2Cl2,9,8.7,9
-CH3Cl,,13,12
-Halon1201,,,376
\ No newline at end of file
+Source,https://www.ghgprotocol.org/sites/default/files/ghgp/Global-Warming-Potential-Values%20%28Feb%2016%202016%29_1.pdf,https://www.ghgprotocol.org/sites/default/files/ghgp/Global-Warming-Potential-Values%20%28Feb%2016%202016%29_1.pdf,https://www.ghgprotocol.org/sites/default/files/ghgp/Global-Warming-Potential-Values%20%28Feb%2016%202016%29_1.pdf,"https://www.ipcc.ch/site/assets/uploads/2018/07/WGI_AR5.Chap_.8_SM.pdf, Table 8.SM.16 and https://www.ipcc.ch/site/assets/uploads/2018/02/WG1AR5_Chapter08_FINAL.pdf , Table 8.7 (page 714)."
+OpenSCMMetricName,SARGWP100,AR4GWP100,AR5GWP100,AR5CCFGWP100
```
Comment (znicholls) [inline src/openscm_units/data/metric_conversions.csv:2]: Is `AR5CCFGWP100` a standard name? If no, can we add some more detail in the source e.g. a comment like "AR5CCFGWP100 is AR5GWP100 including climate-carbon cycle feedbacks"?

File: CHANGELOG.rst (hunk 2 above)
Comment (JGuetschow) [reply CHANGELOG.rst:6]: The name in the IPCC report is actually "... with climate-carbon feedbacks" without "cycle" (which I find a bit weird) but I just used the "official" name.

File: src/openscm_units/data/metric_conversions.csv (hunk 3 above)
Comment (JGuetschow) [reply src/openscm_units/data/metric_conversions.csv:2]: I don't think it's a standard name. The acronym for climate-carbon cycle feedbacks used in AR5 for GWPs is "cc fb" (AR5 WG1, Table 8.7). AR5CCF is used in the PRIMAP emissions module, so I kept that.

Comment (JGuetschow) [reply src/openscm_units/data/metric_conversions.csv:2]: I've added a comment

File: src/openscm_units/data/metric_conversions.csv [hunk 4]
```diff
@@ -1,92 +1,92 @@
...
+HFC41,150,,116,141
+HFC125,2800,3500,3170,3691
+HFC134,1000,,1120,1337
+HFC134a,1300,1430,1300,1549
+HFC143,300,,328,397
+HFC143a,3800,4470,4800,5508
+HFC152,,,16,20
+HFC152a,140,124,138,167
+HFC161,,,4,4
+HFC227ea,2900,3220,3350,3077
```
[the remark this answers is not in the document]
Comment (JGuetschow) [reply src/openscm_units/data/metric_conversions.csv:38]: correct

File: src/openscm_units/data/metric_conversions.csv [hunk 5]
```diff
@@ -1,92 +1,92 @@
...
+HFE236fa,,,979,1177
+HFE245fa1,,,828,997
+HFE263fb2,,,1,2
+HFE329mcc2,,,3070,3598
+HFE338mcf2,,,929,1118
+HFE347mcf2,,,854,1028
+HFE356mec3,,,387,468
+HFE356pcf2,,,719,867
+HFE356pcf3,,,446,540
+HFE365mcf3,,,,1
```
Comment (znicholls) [inline src/openscm_units/data/metric_conversions.csv:86]: Is there no AR5GWP100 available?

**Comment by znicholls** [comment]
One question, otherwise looks good

File: src/openscm_units/data/metric_conversions.csv (hunk 5 above)
Comment (JGuetschow) [reply src/openscm_units/data/metric_conversions.csv:86]: It's below 1 and was empty before I started editing the file, so I kept it that way. I didn't add any substances.

Comment (znicholls) [reply src/openscm_units/data/metric_conversions.csv:86]: Ok it seems odd to only have a 'with carbon cycle feedbacks' number but it's not worth losing sleep over.

**Comment by JGuetschow** [comment]
Integrated Mika's fixes.

File: src/openscm_units/data/metric_conversions.csv (hunk 5 above)
Comment (JGuetschow) [reply src/openscm_units/data/metric_conversions.csv:86]: the non-CCF number is "<1" in AR5. We could just put 1 instead of effectively using 0.

Comment (znicholls) [reply src/openscm_units/data/metric_conversions.csv:86]: I'm also happy with that (would probably be my preference to be honest)

**Review by znicholls — APPROVED** [review]
lgtm once you decide whether to include HFE365mcf3 without feedbacks or not

**Comment by JGuetschow** [comment]
I added 1 as AR5GWP for HFE-365mcf. There is no AR4 GWP though the german "Umweltbundesamt" says AR4 GWP100 is 11 citing the IPCC AR4: [UBA](https://www.umweltbundesamt.de/sites/default/files/medien/2503/dokumente/treibhauspotentiale_ausgewaehlter_verbindungen_und_deren_gemische.pdf)
As I could not verify that, I left the AR4 column empty.

**Comment by znicholls** [comment]
Nice. I'll leave to @mikapfl for a final look and merge if they're happy

**Review by mikapfl — APPROVED** [review]

**Comment by mikapfl** [comment]
Unless I'm missing something, I can't merge anything. Github tells me "Only those with write access to this repository can merge pull requests."

**Comment by znicholls** [comment]
Oops my bad must have set settings differently to what I remember

**Pull request merged**